pkg/armhelpers/storage.go

```diff
@@ -49,3 +50,21 @@ func (as *AzureStorageClient) DeleteBlob(vhdContainer, vhdBlob string) error {
 
 	return blobRef.Delete(&azStorage.DeleteBlobOptions{})
 }
+
+// CreateContainer creates the CloudBlobContainer if it does not exist
+func (as *AzureStorageClient) CreateContainer(containerName string) (bool, error) {
+	bs := as.client.GetBlobService()
```

Comment:
Lines 56 & 57 are duplicated. Can we move them to a separate method?

Reply:
Thanks for the reviewing, I removed the duplicate lines